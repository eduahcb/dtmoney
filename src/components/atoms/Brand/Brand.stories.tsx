import React, { ReactElement } from 'react'

import { Brand } from 'components/atoms/Brand'

export default {
  title: 'Components/Brand',
  component: Brand
}

export const usage = (): ReactElement => (
  <div style={{ backgroundColor: 'var(--blue)', padding: '8px' }}>
  
  </div>
)
