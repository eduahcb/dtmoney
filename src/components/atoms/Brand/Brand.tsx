import React, { ReactElement } from 'react'

import Logo from 'assets/logo.svg'

export const Brand = (): ReactElement => {
  return (
    <img src={Logo} alt="dt money logo" />
  )
}
