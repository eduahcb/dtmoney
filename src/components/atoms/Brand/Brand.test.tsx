import React from 'react'
import { render, screen } from '@testing-library/react'

import { Brand } from 'components/atoms/Brand'

describe('Brand component', () => {
  test('render component', () => {
    render(<Brand />)

    expect(screen.getByAltText('dt money logo')).toBeInTheDocument()
  })

  test('match snapshot', () => {
    const { asFragment } = render(<Brand />)

    expect(asFragment()).toMatchSnapshot()
  })
})
