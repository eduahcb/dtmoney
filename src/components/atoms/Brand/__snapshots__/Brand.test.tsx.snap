// Vitest Snapshot v1

exports[`Brand component > match snapshot 1`] = `
<DocumentFragment>
  <img
    alt="dt money logo"
    src="/src/assets/logo.svg"
  />
</DocumentFragment>
`;
